client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -50,6 +52,27 @@ type EmailProvidersStackedComparisonProps = {
 	gSuiteAnnualProduct?: any;
 };
 
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const noop = () => {};
+
+const ProfessionalEmailComparisonObject: ProviderComparison = {
+	header: <h1> Professional Email </h1>,
+	tools: translate( 'Integrated email management, Inbox, calendar and contacts' ),
```

Comment:
Would it be better to see these texts as a list of features and pass an array instead? It might be easier to wrangle and display then.

Reply:
I think this way is safer in the sense of providing what we need to provide for each case, the component will just read that object and place everything in place and in the right order.

Reply:
Let's leave it like this and see if it fits good with what I've designed in the next PR 🙂